Die Zeit.js

```diff
@@ -169,32 +144,40 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "newspaperArticle",
-				"creators": [],
-				"notes": [],
-				"tags": [
-					"Muammar al-Gaddafi",
-					"Muammar al-Gaddafi",
-					"Mustafa Abdel Dschalil",
-					"Stadt",
-					"Bani Walid",
-					"Berg"
+				"title": "Libyen: Rebellen bereiten Angriff auf Bani Walid vor",
+				"creators": [
+					{
+						"firstName": "",
+						"lastName": "AFP",
+						"creatorType": "author"
```

Comment:
I'd be inclined to say we don't want the press agencies as authors. I don't think they're commonly cited as such and I believe their wires are typically re-written by staff writers, so they're not authors in the technical sense. Is that doable?

If there's no way to distinguish them from "real" authors, I'd want `"fieldMode": 1`